Kconfig.zephyr

```diff
@@ -623,6 +623,12 @@ config MCUBOOT_EXTRA_IMGTOOL_ARGS
 	  you can use this option to pass extra options to imgtool.
 	  For example, you could set this to "--version 1.2".
 
+config MCUBOOT_GENERATE_UNSIGNED_IMAGE
+	bool "Generate unsigned binary image bootable with MCUboot"
+	help
+	  The unsigned binary image will be generated even if the MCUboot signing key is
+	  not provided.
```

Comment:
and if the signing key is provided, and `MCUBOOT_GENERATE_UNSIGNED_IMAGE=y` at the same time, what happens then ?

This is not clear to the reader.
As it has been decided to keep a dedicated setting for this, then the `MCUBOOT_SIGNATURE_KEY_FILE` setting should have a `depends on !MCUBOOT_GENERATE_UNSIGNED_IMAGE` to ensure a user don't specify both.

Should the

Reply:
Updated help statement and added dependency as suggested